crates/runtime/src/builder.rs

```diff
@@ -117,6 +119,13 @@ impl RuntimeBuilder {
         self
     }
 
+    /// Configures a separate Tokio runtime instance to be used by the servers.
+    /// (e.g. HTTP, Flight, etc.)
+    pub fn with_tokio_servers_runtime(mut self) -> std::io::Result<Self> {
+        self.tokio_servers_runtime = Some(Arc::new(TokioRuntime::new()?));
```

Comment:
This is causing a panic on shutdown currently:

```output
thread 'main' panicked at /Users/phillip/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.40.0/src/runtime/blocking/shutdown.rs:51:21:
Cannot drop a runtime in a context where blocking is not allowed. This happens when a runtime is dropped from within an asynchronous context.
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
2024/10/24 20:45:31 ERROR error running Spice.ai error="exit status 101"
```

Reply:
@phillipleblanc  - for learning purpose, does it affect how we process queries / does query execution for flight requests is running in this new runtime vs no?

Reply:
No it should be transparent to the query execution.

Reply:
Verified with the new approach I don't get this panic anymore.